app/code/core/Mage/Adminhtml/Block/Sales/Order/Create/Comment.php

```diff
@@ -51,7 +51,7 @@ public function getHeaderText()
      */
     public function getCommentNote()
     {
-        return $this->escapeHtml($this->getQuote()->getCustomerNote());
+        return $this->escapeHtml((string)$this->getQuote()->getCustomerNote());
```

Comment:
```suggestion
        return $this->escapeHtml($this->getQuote()->getCustomerNote());
```
Revert after https://github.com/OpenMage/magento-lts/pull/2884